src/Microsoft.Health.Fhir.SqlServer.UnitTests/IndentedStringBuilderTests.cs

```diff
@@ -5,6 +5,7 @@
 
 using System;
 using System.Text;
+using Microsoft.Health.SqlServer;
```

Comment:
Should this test be moved now that IndentedStringBuilder is part of Microsoft.Health.SqlServer

Reply:
Yes, along with `EnumerableExtensionsTests.cs`